### PR TITLE
ci: add node.js 12 to CircleCI

### PR DESCRIPTION
**Pull request by faustbrian**
## What kind of change does this PR introduce? (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build-related change
- [ ] Documentation
- [ ] Code style update
- [x] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change? (check one)

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by ghost** [comment]
Your pull request doesn't follow our [contribution guidelines](https://docs.ark.io/guidebook/contribution-guidelines/). Please review and correct it.

**Comment by ghost** [comment]
The [ci/circleci: test-node-11](https://circleci.com/gh/ArkEcosystem/explorer/830?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link) job is failing as of [e49f902b744473e2a075c4c272e0eff9f4e1219e](https://github.com/ArkEcosystem/explorer/commit/e49f902b744473e2a075c4c272e0eff9f4e1219e). Please [review the logs for more information](https://circleci.com/gh/ArkEcosystem/explorer/830?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link).

_Once you've pushed the fixes, the build will automatically re-run. Thanks!_

**Review by ItsANameToo — APPROVED** [review]

File: resources/views/livewire/delegate-table.blade.php [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
+            </span>
+        </x-loading.visible>
+    </span>
 
     @if($this->state['status'] === 'active')
-        <x-loading.hidden>
-            <x-tables.desktop.delegates.active :delegates="$delegates" />
-            <x-tables.mobile.delegates.active :delegates="$delegates" />
-        </x-loading.hidden>
+        <div wire:poll.{{ Network::blockTime() }}s="$refresh" wire:key="poll_active_delegates_skeleton">
```
Comment (faustbrian) [inline resources/views/livewire/delegate-table.blade.php]: Why the `$refresh`? Just `wire:poll.{{ Network::blockTime() }}s` should achieve the same.

Comment (alexbarnsley) [reply resources/views/livewire/delegate-table.blade.php]: Good to know, thanks!

**Pull request merged**